java/server/src/org/openqa/selenium/grid/web/Values.java

```diff
@@ -34,7 +35,7 @@
 public class Values {
 
   private static final Json JSON = new Json();
-  private static final ErrorCodec ERRORS = ErrorCodec.createDefault();
+  private static final org.openqa.selenium.remote.ErrorCodec ERRORS = ErrorCodec.createDefault();
```

Comment:
Fully qualified class name is not needed.

Reply:
Fixed